lib/ui/mcnd_app.dart

```diff
@@ -21,6 +22,10 @@ class McndApp extends StatelessWidget {
       ),
     );
 
+    SystemChrome.setPreferredOrientations([
+      DeviceOrientation.portraitUp,
+    ]);
+
```

Comment:
Why not in main initialization function, this doesn't need to be called every time the build function runs